specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json

```diff
@@ -749,6 +749,9 @@
         }
       },
       "type": "object",
+      "required": [
```

Comment:
What are the compatibility implications of making this required? I assume the service was already always populating this?

Reply:
@jonpayne  for context.

I believe these were added to prevent the client from generating these as nullable fields. The server is always populating these so there should be no impact to the customer.

The PR validation is actually failing because of this as well, and it would be great if we can bypass that

Reply:
/cc @lmazuel for overrides

Reply:
@lmazuel can you take a look at this PR? We want to unblock our customers who need access to the new event

Reply:
> I believe these were added to prevent the client from generating these as nullable fields. The server is always populating these so there should be no impact to the customer.

These types have already been released in GA libraries. If we change the swagger to mark these as required, we would need to just add swagger transforms to make them unrequired anyway to avoid API breaks. Which client is this change helping?